### ISSUE TITLE
Add electron support

### ISSUE THREAD

**Issue by nampdn** [benorama/ngrx-demo-apps#7]
It would be nice if this project can have electron support for 3 other desktop platform.

**Comment by benorama** [comment]
Indeed, it would be great, unfortunately, we don't use/know Electron.
Pull request welcome ;)
